projects/packages/forms/src/blocks/contact-form/edit.js

```diff
@@ -114,48 +114,41 @@ function JetpackContactFormEdit( { name, attributes, setAttributes, clientId, cl
 		block => block.name === 'jetpack/button'
 	);
 
-	const {
-		postTitle,
-		canUserInstallPlugins,
-		hasAnyInnerBlocks,
-		postAuthorEmail,
-		selectedBlockClientId,
-		onlySubmitBlock,
-	} = useSelect(
-		select => {
-			const { getBlocks, getBlock, getSelectedBlockClientId, getBlockParentsByBlockName } =
-				select( blockEditorStore );
-			const { getEditedPostAttribute } = select( editorStore );
-			const selectedBlockId = getSelectedBlockClientId();
-			const selectedBlock = getBlock( selectedBlockId );
-			let selectedStepBlockId = selectedBlockId;
-
-			if ( selectedBlock && selectedBlock.name !== 'jetpack/form-step' ) {
-				selectedStepBlockId = getBlockParentsByBlockName(
-					selectedBlockId,
-					'jetpack/form-step'
-				)[ 0 ];
-			}
-
-			const { getUser, canUser } = select( coreStore );
-			const innerBlocksData = getBlocks( clientId );
-
-			const title = getEditedPostAttribute( 'title' );
-			const authorId = getEditedPostAttribute( 'author' );
-			const authorEmail = authorId && getUser( authorId )?.email;
+	const { postTitle, hasAnyInnerBlocks, postAuthorEmail, selectedBlockClientId, onlySubmitBlock } =
+		useSelect(
+			select => {
+				const { getBlocks, getBlock, getSelectedBlockClientId, getBlockParentsByBlockName } =
+					select( blockEditorStore );
+				const { getEditedPostAttribute } = select( editorStore );
+				const selectedBlockId = getSelectedBlockClientId();
+				const selectedBlock = getBlock( selectedBlockId );
+				let selectedStepBlockId = selectedBlockId;
+
+				if ( selectedBlock && selectedBlock.name !== 'jetpack/form-step' ) {
+					selectedStepBlockId = getBlockParentsByBlockName(
+						selectedBlockId,
+						'jetpack/form-step'
+					)[ 0 ];
+				}
 
-			return {
-				postTitle: title,
-				canUserInstallPlugins: canUser( 'create', 'plugins' ),
-				hasAnyInnerBlocks: innerBlocksData.length > 0,
-				postAuthorEmail: authorEmail,
-				selectedBlockClientId: selectedStepBlockId,
-				onlySubmitBlock:
-					innerBlocksData.length === 1 && innerBlocksData[ 0 ].name === 'jetpack/button',
-			};
-		},
-		[ clientId ]
-	);
+				const { getUser } = select( coreStore );
+				const innerBlocksData = getBlocks( clientId );
+
+				const title = getEditedPostAttribute( 'title' );
+				const authorId = getEditedPostAttribute( 'author' );
+				const authorEmail = authorId && getUser( authorId )?.email;
+
+				return {
+					postTitle: title,
+					hasAnyInnerBlocks: innerBlocksData.length > 0,
+					postAuthorEmail: authorEmail,
+					selectedBlockClientId: selectedStepBlockId,
+					onlySubmitBlock:
+						innerBlocksData.length === 1 && innerBlocksData[ 0 ].name === 'jetpack/button',
+				};
+			},
+			[ clientId ]
+		);
```

Comment:
If you are seeing a lot of changes here, try hiding white space on the diff. It's easier to read. We just remove canUserInstallPlugins here since we are using the feature flag instead of this permissions check.